deps/verifier/src/tdx/claims.rs

```diff
@@ -15,7 +15,7 @@ use serde_json::{Map, Value};
 use thiserror::Error;
 
 use super::quote::Quote;
-use crate::{eventlog::AAEventlog, tdx::quote::QuoteV5Body, TeeEvidenceParsedClaim};
+use crate::{tdx::quote::QuoteV5Body, TeeEvidenceParsedClaim};
 use ::eventlog::CcEventLog;
```

Comment:
After removing eventlog mod.rs this can be simplified:
```suggestion
use eventlog::CcEventLog;
```
Additional `::` was added to point to external module different from available in scope (all similar occurrences can be also updated)

Reply:
Thanks, fixed.